target/linux/ramips/mt7621/config-4.14

```diff
@@ -284,3 +284,7 @@ CONFIG_WEAK_ORDERING=y
 CONFIG_XPS=y
 CONFIG_ZLIB_DEFLATE=y
 CONFIG_ZLIB_INFLATE=y
+CONFIG_REGULATOR=y
+CONFIG_REGULATOR_GPIO=y
+CONFIG_REGULATOR_FIXED=y
```

Comment:
This config symbol doesn't exists. Do you mean `CONFIG_REGULATOR_FIXED_VOLTAGE` by any chance? Furthermore you broke alphabetical order here. Please consider using `make kernel_menuconfig CONFIG_TARGET=subtarget` to add kernel config symbols.

I'm quite sure  `REGULATOR_GPIO` isn't required. It's meant for regulators which set different voltage levels based on the GPIO (input) value.

Am I right that the PCIe vcc is enabled by default by the bootloader? At least with this code, OpenWrt never enabled them.

Reply:
Uh, thanks for pointing that out. Yes, the bootloader enables the PCIe power GPIOs because mt7621-pci driver lacks hotplug support. Slots where no device is detected on bootup end up permanently disabled.